schedule/migration/aarch64_regression_test_offline.yaml

```diff
@@ -59,6 +61,10 @@ conditional_schedule:
     REGRESSION_SERVICE:
       1:
         - installation/install_service
+  openldap_to_389ds:
+    REGRESSION_389DS:
```

Comment:
i suggest do not use this flag since all platform should support this package by default.

Reply:
This is only used for aarch64_regression_test_offline,  Each yaml file has added this flag. And we only run one/two cases at one platform. So the case which add this setting will run 389s module

Reply:
YWe add this flag just help owner to be able to control which job will run 389ds since we don't want to run this on all test except x86.